src/dtlstransport.cpp

```diff
@@ -149,7 +147,7 @@ void DtlsTransport::postHandshake() {
 
 void DtlsTransport::runRecvLoop() {
 	const size_t maxMtu = 4096;
-
+    postCreation();
```

Comment:
Good catch but the fix seems unsafe. Since it's called in the thread, there is no guarantee it'll be called after `DtlsSrtpTransport` is initialized.

Reply:
Absolutely. This is probably why I am currently having a 50% fail rate at starting the transport successfully...

Reply:
This should be reliably fixed by #147